packages/ai-jsx/src/batteries/sidekick/platform/sidekick.tsx

```diff
@@ -131,8 +126,8 @@ export function Sidekick(props: SidekickProps) {
             userProvidedGenUIUsageExamples={props.genUIExamples}
             userProvidedGenUIComponentNames={props.genUIComponentNames}
           />
-          {props.systemMessage}
           <ConversationHistory />
+          {props.systemMessage}
```

Comment:
I think rather than doing this and/or in addition to it, you want to remove the `finalSystemMessageBeforeResponse` prop.

@petersalas I notice that `finalSystemMessageBeforeResponse` is only actually used in the `functionResponse` case. In the original Sidekick implementation, that made sense, because we were trying to force the LLM to use a `Card`. 

But now that we're being more generic and/or not using cards, maybe we should kill `finalSystemMessageBeforeResponse` entirely and just do what @zkoch is doing here?

Reply:
Seems reasonable to me.